### ISSUE TITLE
Ux doesn't load after 11.18 Patch update

### ISSUE THREAD

**Issue by VihagChaturvedi** [nhubaotruong/league-of-legends-linux-garena-script#12]
I have been having some issues after the new 11.18 Patch

I have posted a support request on the r/leagueoflinux

[here is the link to it where i have provided more info](https://www.reddit.com/r/leagueoflinux/comments/po7io4/potentially_new_issue_ux_doesnt_fully_load_and/hcufg21/?context=3)

info includes, logs, device info, screenshot of the issue, fixes attempted, etc.

TL;DR: The UX is stuck on the loading screen. Specifically after the 11.18 Patch

**Comment by VihagChaturvedi** [comment]
Update: apparently the problem lies with Wine LoL and Garena servers

[Source](https://www.reddit.com/r/leagueoflinux/comments/po7io4/potentially_new_issue_ux_doesnt_fully_load_and/hcwp342?utm_medium=android_app&utm_source=share&context=3)

**Comment by nhubaotruong** [comment]
Sorry I cannot check issue for now, I'm in a limited connection area and have to use mobile internet. I cannot even update the game :disappointed: . Sorry I cannot help you now

**Comment by VihagChaturvedi** [comment]
Ah np np.

I can wait lol. Just had to inform u regarding the issue so that u aware.

If u are on a trip, have fun!

**Comment by sangowen** [comment]
Hey, I am one of the Garena players that failed to fully load `LeagueClientUX` after 11.18.

I've logged in Windows and checked what's the difference between the logs.

It turns out that `--garena-plus-token=` was not supposed to be passed to `LeagueClient/LeagueClient.exe`. It was passed to its parent process `Riot Client/RiotClientServices.exe`.

By changing the `League Of Legends` -> `Game options` -> `Executable` to `[home]/garena/drive_c/Garena/Games/32775/Riot Client/RiotClientServices.exe` (replace [home] to linux user home dir), I managed to once again login to League of Legends. It feels like the client got heavier and took much longer to load than before. `RiotClientService` also seems to launch crash reporters and stuff.

I think in the past we were somehow bypassing `RiotClientService.exe` and still got the game launched regardless. I don't know if this was intended by @nhubaotruong , but this seems to not work anymore.

Hope this can be reproduced by fellow Garena players who have problem launching the game since 11.18.
